tests/pr_test.go

```diff
@@ -31,35 +41,58 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
-	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:      t,
-		TerraformDir: dir,
-		Prefix:       prefix,
-		// only one `lite` instance can be provisioned for each RG. Always create a new RG.
-		// ResourceGroup: resourceGroup,
-		TerraformVars: map[string]interface{}{
-			"access_tags": permanentResources["accessTags"],
+func TestDAInSchematics(t *testing.T) {
+	t.Parallel()
+
+	var region = validRegions[rand.IntN(len(validRegions))]
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "wp-da",
+		TarIncludePatterns: []string{
+			"*.tf",
+			standardSolutionDir + "/*.tf",
 		},
-		CheckApplyResultForUpgrade: true,
+		ResourceGroup:          resourceGroup,
+		TemplateFolder:         standardSolutionDir,
+		Tags:                   []string{"test-schematic"},
+		DeleteWorkspaceOnFail:  false,
+		WaitJobCompleteMinutes: 60,
 	})
-	return options
-}
 
-func TestRunBasicExample(t *testing.T) {
-	t.Parallel()
-
-	options := setupOptions(t, "scc-wp", basicExampleDir)
+	options.TerraformVars = []testschematic.TestSchematicTerraformVar{
+		{Name: "ibmcloud_api_key", Value: options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"], DataType: "string", Secure: true},
+		{Name: "resource_group_name", Value: options.Prefix, DataType: "string"},
+		{Name: "region", Value: region, DataType: "string"},
+		{Name: "scc_workload_protection_instance_tags", Value: options.Tags, DataType: "list(string)"},
+		{Name: "scc_workload_protection_resource_key_tags", Value: options.Tags, DataType: "list(string)"},
+		{Name: "scc_workload_protection_access_tags", Value: permanentResources["accessTags"], DataType: "list(string)"},
+		{Name: "prefix", Value: options.Prefix, DataType: "string"},
+	}
 
-	output, err := options.RunTestConsistency()
+	err := options.RunSchematicTest()
 	assert.Nil(t, err, "This should not have errored")
-	assert.NotNil(t, output, "Expected some output")
 }
 
-func TestRunAdvancedUpgradeExample(t *testing.T) {
+func TestRunUpgradeDA(t *testing.T) {
 	t.Parallel()
 
-	options := setupOptions(t, "scc-wp-upg", advancedExampleDir)
+	var region = validRegions[rand.IntN(len(validRegions))]
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
```

Comment:
Can we use the new schematics upgrade test instead? All DA tests should use schematics